mem0/configs/embeddings/base.py

```diff
@@ -10,9 +10,13 @@ def __init__(
         self,
         model: Optional[str] = None,
         embedding_dims: Optional[int] = None,
+        api_key: Optional[str] = None,
 
         # Ollama specific
-        base_url: Optional[str] = None
+        ollama_base_url: Optional[str] = None,
+
+        # LM Studio specific
+        lmstudio_base_url: Optional[str] = "http://localhost:1234/v1"
```

Comment:
Is 1234 a random number or is it the actual port number? If it is a random number then you should mention something like

```
lmstudio_base_url: Optional[str] = "http://localhost:1234/v1 # here add your localhost and port
```
Else, it is good.

Reply:
No it's not random, this is base URL taken from LM Studio itself.